scripts/coreboot-sdk.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: GPL-3.0-only
+
+# Build the coreboot toolchains
+
+# shellcheck disable=SC1091
+
+set -e
+
+. /etc/os-release
+if [ "$ID" = "arch" ] || [[ "$ID_LIKE" =~ "arch" ]]; then
+    sudo pacman -S --noconfirm \
+        bison \
+        bzip2 \
+        ca-certificates \
+        curl \
+        flex \
+        gcc \
+        gcc-ada \
+        make \
+        patch \
+        tar \
+        xz \
+        zlib
+elif [ "$ID" = "fedora" ] || [[ "$ID_LIKE" =~ "fedora" ]]; then
+    sudo dnf install --assumeyes \
+        bison \
+        bzip2 \
+        ca-certificates \
+        curl \
+        flex \
+        gcc \
+        gcc-c++ \
+        gcc-gnat \
+        make \
+        patch \
+        tar \
+        xz \
+        zlib-devel
+elif [ "$ID" = "ubuntu" ] || [[ "$ID_LIKE" =~ "debian" ]]; then
+    sudo apt-get --quiet update
+    sudo apt-get --quiet install --no-install-recommends --assumeyes \
```

Comment:
I believe `--assumeyes` should be `--assume-yes` with apt-get.

Reply:
Fixed.